plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -264,7 +264,14 @@ def helper_usable?(helper)
   def run_helper_inspection(helper, filter, do_extract, file_store_tmp, file_store_final, scope)
     begin
       helper.inject_helper
-      helper.run_helper(scope)
+      if !helper.is_supported?
+        raise Machinery::Errors::UnsupportedHelperVersion.new(
+          "Error: machinery-helper is not compatible with the Machinery version" \
+          " #{Machinery::VERSION}. The versions need to match."
+        )
+      else
```

Comment:
The `else` branch is not needed. It would look cleaner like that imo:

```ruby
helper.inject_helper

if !helper.is_supported?
  raise Machinery::Errors::UnsupportedHelperVersion.new(
    "Error: machinery-helper is not compatible with the Machinery version" \
    " #{Machinery::VERSION}. The versions need to match."
  )
end

helper.run_helper(scope)
```